### ISSUE TITLE
How to use a local external storage with snap’s Nextcloud?

### ISSUE THREAD

**Issue by NightMachinary** [nextcloud/nextcloud-snap#822]
I understand that snap isolates the installation; But I need to disable this for [local](url
https://docs.nextcloud.com/server/12/admin_manual/configuration_files/external_storage/local.html) external storages to work. What steps should I take?

I want to add ~/Downloads as a local storage. What permissions should I set on this folder? I’m ok with doing a ‘chmod a+rw -R’ on it, since I am the sole user.

**Comment by kyrofa** [comment]
You can try bind-mounting your downloads folder into `/media/` somewhere and connecting the `removable-media` interface.

**Comment by Chrozayis** [comment]
I have posted a related issue under this thread [Issue#797](https://github.com/nextcloud/nextcloud-snap/issues/767)

**Comment by kyrofa** [comment]
I'm assuming this issue can be closed. Feel free to comment or re-open if necessary.

**Comment by nightmare1942** [comment]
Hi,
I do have a similar problem with NC14.0.5.
I mounted an SMB share via fstab to /media/storage.
Afterwards I tried to use the external storage (local) extension to mound a folder under this mountpoint.
I worked for a long time but now it has the following error:

`stat(): stat failed for /media/storage/pictures/ at /snap/nextcloud/10791/htdocs/lib/private/Files/Storage/Local.php#142`

I also connected the `removable-media` plug as described [here](https://github.com/nextcloud/nextcloud-snap#removable-media)

But I still only see the red dot and the media is not working.

Any clues?
